Source/astcenc_weight_align.cpp

```diff
@@ -156,21 +152,25 @@ static void compute_angular_offsets(
 		const float *sinptr = sin_table[isample];
 		const float *cosptr = cos_table[isample];
 
-		for (int j = 0; j < max_angular_steps; j++)
+		vfloat sample_weightv(sample_weight);
+		for (int j = 0; j < max_angular_steps; j += ASTCENC_SIMD_WIDTH) // arrays are multiple of SIMD width (ANGULAR_STEPS), safe to overshoot max
```

Comment:
Can we add a static_assert for the ANGULAR_STEPS being a multiple of SIMD_WIDTH.

Reply:
Done in 86c1dd8